demo.postman_collection.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
looks like leftover

Reply:
nope it's to try via postman how it works